website/client/src/components/groups/tavern.vue

```diff
@@ -513,10 +513,15 @@
     padding: 0em;
 
     .below-header-sections {
-      padding: 1em 1.75em 1em 1.75em;
+      padding: 1em 24px 1em 24px;
     }
   }
 
+  .px-3 {
```

Comment:
instead of changing the rules for this class, can't we simply use `px-4`which has 24px paddings

Reply:
Oh I wasn't aware! I'll make those updates when I have a minute!

Reply:
thanks! those classes come from Bootstrap's utilities https://getbootstrap.com/docs/4.3/utilities/spacing/

Reply:
@paglias finally had some time to get these updated